extensions/amp-analytics/0.1/vendors.js

```diff
@@ -707,7 +707,7 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
           '${baseSuffix}',
       'error': '${host}/collect?${basePrefix}&' +
           't=exception&' +
-          'exd=${errorMessage}' +
+          'exd=name=${errorName},msg=${errorMessage}' +
```

Comment:
I don't know how we want to config GA here. but I think definitely no ',' 
maybe `exd=${errorName}-${errorMessage}` ?
cc @avimehta

Reply:
It'll be nice if you can move the value out into a variable. That way, people can override the variable and use their own format if they want.

Otherwise, prescribing a format is fine...

Reply:
how about `en=${errorName}&em=${errorMessage}`?